frontend/integration-tests/tests/crd-extensions.scenario.ts

```diff
@@ -131,7 +131,7 @@ describe('CRD extensions', () => {
         });
 
         it(`displays the ${crd} instance in the ${dropdownMenuName}`, async () => {
-          await browser.get(`${appHost}`);
+          await browser.get(`${appHost}/dashboards`);
```

Comment:
This will break the tests on native k8s. I'd revert this change.

Reply:
Ah, native K8s, hmm. Somehow I needed it for Firefox. Will dig around...

Reply:
Maybe you need the trailing slash? You had that other places.

Reply:
I tried `/`, `/#`, `/k8s/cluster/projects` and none of them work with FF. Any suggestions?

Reply:
When you say it doesn't work, what exactly happens? We have this in a few other places, so I'm not sure why this one would be different.

Reply:
Actually this whole thing may just be the flakes. I went back and retried with trailing `/`, and it failed like before, I'm pretty sure the error is exactly the same, but this time it passed for "help menu" and failed for "user menu."
```
Failed Specs:

1) CRD extensions ConsoleLink CRD : displays the ConsoleLink instance in the user menu
   WebDriverError: Reached error page: about:neterror?e=connectionFailure&u=http%3A//localhost%3A9000/&c=UTF-8&f=regular&d=Firefox%20can%E2%80%99t%20establish%20a%20connection%20to%20the%20server%20at%20localhost%3A9000.
       at Object.throwDecodedError /git/console/frontend/node_modules/selenium-webdriver/lib/error.js:514:15
       at parseHttpResponse /git/console/frontend/node_modules/selenium-webdriver/lib/http.js:519:13
       at doSend.then.response /git/console/frontend/node_modules/selenium-webdriver/lib/http.js:441:30
       at process._tickCallback internal/process/next_tick.js:68:7
   From: Task: WebDriver.navigate().refresh()
       at Driver.schedule /git/console/frontend/node_modules/selenium-webdriver/lib/webdriver.js:807:17
       at Navigation.refresh /git/console/frontend/node_modules/selenium-webdriver/lib/webdriver.js:1167:25
       at ProtractorBrowser.refresh /git/console/frontend/node_modules/protractor/built/browser.js:783:43
       at /git/console/frontend/integration-tests/tests/crd-extensions.scenario.ts:137:25
       at Generator.next <anonymous>
       at fulfilled /git/console/frontend/integration-tests/tests/crd-extensions.scenario.ts:5:58
       at process._tickCallback internal/process/next_tick.js:68:7
```
This is the first time I see it pass for "help menu" and failed for "user menu." It was always failing at the first come-around (=help menu). Hence, this could be just flake, and FF somehow is more susceptible to that...
Yet, how in the world `.refresh()` would fail often after a successful `.get()`?

Reply:
Running `bridge` on host brought in consistency, reducing my version of flakes, and I was able to verify FF worked fine with the original code.